app/AttributionGenerator.js

```diff
@@ -158,7 +158,9 @@ $.extend( AttributionGenerator.prototype, {
 		// Prefer attribution over author:
 		var $attribution = this._asset.getAttribution();
 		if( $attribution ) {
-			return format === 'html' ? $attribution : $attribution.text();
+			if( $attribution.length && $attribution.text().length ) {
```

Comment:
Sure. It being common does not mean it's good though, especially when talking about JS practices :)

In this particular case I doubt anyone would get confused, though for things less known than the length attribute, such as all own code, it does make a real difference.